### PR TITLE
Revert & Fix "Fix Erroneous Ellipsis Overflow"

### PR DESCRIPTION
**Pull request by wopian**
Reverts hummingbird-me/hummingbird#514

I really need a proper way to test **every** case with CSS changes. Worked for everything but inline @mentions and URLS...which I didn't have in any of my test stories. :astonished:

Pending `squash`

### CODE REVIEW

**Comment by NuckChorris** [comment]
Is this ready?

**Comment by wopian** [comment]
Yes

File: app/assets/stylesheets/partials/status-panel.css.scss [hunk 1]
```diff
@@ -168,17 +168,17 @@
     padding: 0;
     margin-bottom: 0;
     overflow: hidden;
+    text-overflow: ellipsis;
     max-height: 500px;
+    max-width: 99.9%; // HACK
```
Comment (NuckChorris) [inline app/assets/stylesheets/partials/status-panel.css.scss]: Might wanna give details on the hack

File: app/assets/stylesheets/partials/status-panel.css.scss [hunk 2]
```diff
@@ -168,17 +168,17 @@
     padding: 0;
     margin-bottom: 0;
     overflow: hidden;
+    text-overflow: ellipsis;
     max-height: 500px;
+    max-width: 99.9%; // HACK; Prevent ellipsis overflow of characters
```
Comment (NuckChorris) [inline app/assets/stylesheets/partials/status-panel.css.scss]: I am honestly _more_ confused now with this comment than I was before.  @JoshFabian it's up to you if this hack is up to your CSS standards.

**Comment by wopian** [comment]
cc @JoshFabian

**Pull request merged**